onnxruntime/core/framework/TensorSeq.h

```diff
@@ -56,16 +58,29 @@ class TensorSeq {
     return tensors_.cend();
   }
 
-  // Get by index
+  // Get onnxruntime::Tensor by index
   const Tensor& Get(size_t i) const {
+    return GetAt(i).Get<Tensor>();
```

Comment:
>return GetAt(i).Get<Tensor>(); [](http://example.com/codeflow?start=4&length=30)

Now that we have OrtValues, it would be nice to enforce some place that they indeed contain Tensors. Either during construction/population or here.

Reply:
`OrtValue::Get` already has an `ORT_ENFORCE` to ensure that it contains the right type.

Reply:
True. This is at Get() time which makes it hard to trace where this was added.